packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -41,6 +41,8 @@ abstract class RunCommandBase extends FlutterCommand {
         negatable: true,
         defaultsTo: false,
         help: 'Start tracing during startup.');
+    // Hidden flag for passing options to the VM.
+    argParser.addOption('dart-flags', hide:true);
```

Comment:
iirc, --verbose will show this option. So we should still have help text.

Also, space after colon.